VultisigApp/VultisigApp/Extensions/TextField.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  TextField.swift
+//  VultisigApp
+//
+//  Created by Enrique Souza Soares on 30/06/24.
+//
+// This file extension is used to limit the number of characters that can be entered in a TextField.
+
+import Foundation
+import SwiftUI
+import Combine
+
+#if canImport(UIKit)
+import UIKit
+
+private var maxLengthKey: UInt8 = 0
+
+extension UITextField {
+    @IBInspectable var maxLength: Int {
+        get {
+            return objc_getAssociatedObject(self, &maxLengthKey) as? Int ?? 50
+        }
+        set {
+            objc_setAssociatedObject(self, &maxLengthKey, newValue, .OBJC_ASSOCIATION_RETAIN_NONATOMIC)
+            addTarget(self, action: #selector(checkMaxLength), for: .editingChanged)
+        }
+    }
+    
+    @objc func checkMaxLength() {
+        guard let text = self.text, maxLength > 0 else { return }
+        if text.count > maxLength {
+            self.text = String(text.prefix(maxLength))
+        }
+    }
+}
+#endif
+
+struct MaxLengthModifier: ViewModifier {
+    @Binding var text: String
+    var maxLength: Int = 50
+    
+    func body(content: Content) -> some View {
+        content
+            .onChange(of: text) { newValue in
```

Comment:
Fix warning please 
`'onChange(of:perform:)' was deprecated in iOS 17.0: Use `onChange` with a two or zero parameter action closure instead.`

Reply:
Ah, sure